packages/trader/src/AppV2/Components/Guide/guide.tsx

```diff
@@ -20,11 +21,30 @@ const Guide = observer(({ has_label, show_guide_for_selected_contract }: TGuide)
     } = useStore();
     const { contract_type, is_vanilla } = useTraderStore();
     const contract_type_title = is_vanilla ? CONTRACT_LIST.VANILLAS : getContractTypesConfig()[contract_type]?.title;
+    const { trade_types } = useContractsForCompany();
+    const order = [
+        CONTRACT_LIST.RISE_FALL,
+        CONTRACT_LIST.ACCUMULATORS,
+        CONTRACT_LIST.MULTIPLIERS,
+        CONTRACT_LIST.VANILLAS,
+        CONTRACT_LIST.TURBOS,
+        CONTRACT_LIST.HIGHER_LOWER,
+        CONTRACT_LIST.TOUCH_NO_TOUCH,
+        CONTRACT_LIST.MATCHES_DIFFERS,
+        CONTRACT_LIST.EVEN_ODD,
+        CONTRACT_LIST.OVER_UNDER,
+    ];
```

Comment:
Is this temporary?

Reply:
Yes, I just moved this order array from the child component to the parent.

Reply:
It will be refactored soon in DTRA-2015 card

Reply:
Alrighty, since there's already a card created for the refactor, probably no need to add the same TODO comment 👍